examples/quotes-api/internal/http/graph/schema.resolvers.go

```diff
@@ -0,0 +1,33 @@
+package graph
+
+// This file will be automatically regenerated based on the schema, any resolver implementations
+// will be copied through when generating and any unknown code will be moved to the end.
+// Code generated by github.com/99designs/gqlgen version v0.17.30
+
+import (
+	"context"
+
+	"github.com/google/uuid"
+	"github.com/graphbound/graphbound/examples/quotes-api/internal/http/graph/generated"
+	"github.com/graphbound/graphbound/examples/quotes-api/internal/http/graph/model"
+)
+
+// Quotes is the resolver for the quotes field.
+func (r *queryResolver) Quotes(ctx context.Context) ([]*model.Quote, error) {
+	q, err := r.getQuoteUseCase.Execute(ctx)
+	if err != nil {
+		return nil, err
+	}
+
+	return []*model.Quote{
+		{
+			ID:   uuid.NewString(),
+			Text: q.Value,
+		},
+	}, nil
```

Comment:
this part is not automatically generated right?

Reply:
Correct, this is the actual resolver implementation.